### PR TITLE
Expose SPIRVToOCL12, SPIRVToOCL20 interfaces in SPIRVToOCL.h

### PR DESCRIPTION
**Pull request by Nuullll**
Signed-off-by: Yilong Guo <yilong.guo@intel.com>

### CODE REVIEW

**Comment by Nuullll** [comment]
This "recommit"s #904, which was accidentally "revert"ed by #840

**Review by AlexeySachkov — APPROVED** [review]

File: lib/SPIRV/SPIRVToOCL.h [hunk 1]
```diff
@@ -185,4 +189,142 @@ class SPIRVToOCLLegacy : public ModulePass {
...
+  SPIRVToOCL20Legacy() : SPIRVToOCLLegacy(ID) {
+    initializeSPIRVToOCL20LegacyPass(*PassRegistry::getPassRegistry());
+  }
+  bool runOnModule(Module &M) override;
+  static char ID;
+};
+
 } // namespace SPIRV
+
+#endif // SPIRVTOOCL_H
```
Comment (svenvh) [inline lib/SPIRV/SPIRVToOCL.h:330]: Please fix the missing EOL at the end of the file.

**Review by svenvh — COMMENTED** [review]
LGTM, except for one little issue.

File: lib/SPIRV/SPIRVToOCL.h (hunk 1 above)
Comment (Nuullll) [reply lib/SPIRV/SPIRVToOCL.h]: Thanks, done!

**Comment by Nuullll** [comment]
Ah, I have no idea about the latest build failure on Linux (didn't reproduce on my local machine). I just added an EOL :-p

**Comment by svenvh** [comment]
> Ah, I have no idea about the latest build failure on Linux (didn't reproduce on my local machine). I just added an EOL :-p

Seems unrelated to your change indeed, I also got it on #1030 .  I think we can merge this change.

**Review by svenvh — APPROVED** [review]

**Pull request merged**